.circleci/config.yml

```diff
@@ -0,0 +1,63 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:8.1-browsers
+        environment:
+          - DB_NAME=lunch_test
+          - DB_USER=lunch_test
+          - DB_PASS=
+          - DB_HOST=localhost
+          - SUPERUSER_NAME=test
+          - SUPERUSER_PASSWORD=test
+          - SUPERUSER_EMAIL=test@lunch.pink
+
+      - image: postgres:9.6
+        environment:
+        - POSTGRES_USER=lunch_test
+        - POSTGRES_DB=lunch_test
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      - run:
+          name: dotenv
+          command: touch .env
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "package.json" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-
+
+      # install deps
+      - run:
+          name: npm-install
+          command: npm install
```

Comment:
Might install the wrong versions if you use `npm` here. Any possibility of using `yarn` instead?